.github/workflows/website.yml

```diff
@@ -4,16 +4,18 @@ name: Website
 # for that branch.
 on:
   push:
-    branches:
-      - "*"
-      - "!master"
+    branches-ignore:
+      - master
```

Comment:
Bug fix: This action was only executed for branches without a slash in the name. Hence, `foo/bar` would _not_ be tested in CI.